src/Reducers/UserFormStateReducer.js

```diff
@@ -26,6 +26,8 @@ export function userFormStateReducer(formState, action) {
           ? draftFormState.programs.delete(action.program)
           : draftFormState.programs.add(action.program);
         break;
+      default:
```

Comment:
I know I previously asked for this to be removed but turns out React warns against missing defaults 😅

Reply:
Yeah i added it back in during the merge :)